app/busy_test.go

```diff
@@ -24,10 +23,10 @@ func TestBusySet(t *testing.T) {
 	}
 
 	require.False(t, busy.IsBusy())
+	require.True(t, compareBusyState(t, busy, cluster.Busy))
 
 	busy.Set(time.Millisecond * 100)
 	require.True(t, busy.IsBusy())
```

Comment:
The failure seems to be in L29, not 30. The issue seems to be a congested CI environment that took more than 100ms to execute from L28 to L29. I think our best chance is to bump the set timeout from 100ms to 500ms.

Reply:
Oh right, I think I mixed the failure 😅